README.md

```diff
@@ -136,290 +156,240 @@ interface RestfulProviderProps<T> {
 
 ![Autocompletion on QueryParams](assets/idp.gif)
 
 Please note that the above example was built using our [OpenAPI generator](#code-generation) in order to infer the type of component from the specification and automatically generate the entire type-safe component in a _very_ quick and easy way.
 
-### Mutations with `Mutate`
+### Mutations with `useMutate`
 
-Restful React exposes an additional component called `Mutate`. These components allow sending requests with other HTTP verbs in order to mutate backend resources.
+`restful-react` exposes an additional hook called `useMutate`. These components allow sending requests with other HTTP verbs in order to mutate backend resources.
 
-[![Edit Restful React demos](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/30n66z45mq)
+[![Edit restful-react demos](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/restful-react-demos-vjets)
 
 ```jsx
-const Movies = ({ dispatch }) => (
-  <ul>
-    <Get path="/movies">
-      {(movies, states, actions) =>
-        movies.map(movie => (
-          <li>
-            {movie.name}
-            <Mutate verb="DELETE">
-              {(deleteMovie, { loading: isDeleting }) => (
-                <button onClick={() => deleteMovie(movie.id).then(() => dispatch("DELETED"))} loading={isDeleting}>
-                  Delete!
-                </button>
-              )}
-            </Mutate>
-          </li>
-        ))
-      }
-    </Get>
-  </ul>
-);
+import React from "react";
+import { useGet, useMutate } from "restful-react";
+
+const base = "https://jsonplaceholder.typicode.com";
+
+const ListItem = ({ id, children }) => {
+  const { mutate: del, loading } = useMutate({
+    verb: "DELETE",
+    path: `/posts/`,
+    base,
+  });
+
+  return (
+    <li key={id}>
+      {loading ? (
+        "Deleting..."
+      ) : (
+        <a
+          href="/"
+          onClick={e => {
+            e.preventDefault();
+            del(id).then(() => alert("Deleted successfully. Pretend it got removed from the DOM."));
+          }}
+        >
+          ❌
+        </a>
+      )}
+      &nbsp;{children}
+    </li>
+  );
+};
+
+const MyHugeList = () => {
+  const { data: posts } = useGet({
+    path: "/posts",
+    base,
+  });
+  return (
+    <div>
+      <h1>Posts</h1>
+      <ul>
+        {posts &&
+          posts.map(post => (
+            <ListItem key={post.id} id={post.id}>
+              {post.title}
+            </ListItem>
+          ))}
+      </ul>
+    </div>
+  );
+};
+export default MyHugeList;
 ```
 
-`Mutate` is strongly typed, and provides intelligent autocompletion out of the box, complete with available verbs and other self-documentation.
+`useMutate` is strongly typed, and provides intelligent autocompletion out of the box, complete with available verbs et al.
```

Comment:
```suggestion
`useMutate` is strongly typed, and provides intelligent autocompletion out of the box, complete with available verbs etc.
```

My guess `et al.` is less know  than `etc.` or `and others`.

Reply:
```suggestion
`useMutate` is strongly typed, and provides intelligent autocompletion out of the box, complete with available verbs.
```

Reply:
Let's just drop it.